pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -907,19 +908,22 @@ func (a Adapter) Exec(command []string) error {
 	return a.ExecuteCommand(componentInfo, command, true, nil, nil)
 }
 
-func (a Adapter) createDockerConfigSecret(parameters common.BuildParameters) error {
-	data, err := utils.CreateDockerConfigDataFromFilepath(parameters.DockerConfigJSONFilename)
+func (a Adapter) createDockerConfigSecret(DockerConfigJSONFilename string, secretName string, nameSpace string) error {
```

Comment:
Formal parameter name (DockerConfigJSONFilename) should start with lower case (e.g. dockerConfigJSONFilename)